### PR TITLE
Services map support for traces and fields mapping sample

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
added a service mapping support for both the traces correlation between 
 - Metrics.serviceName <-> services.name
 - Traces.serviceName <-> services.name
 
This will allow for simple and common integrations of data-prepper proprietary schemas

This PR is also addressing the fields mapping aliasing for similar semantic  meaning fields that have different names - 

### Issues Resolved
- https://github.com/opensearch-project/observability/issues/1473

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: integrations/service-map/samples/preloaded/img/nginx-dashboard.png
Comment (Swiddis) [inline integrations/service-map/samples/preloaded/img/nginx-dashboard.png:1]: Invalid image?

File: integrations/nginx/info/fluet-bit/README.md
Comment (Swiddis) [inline integrations/nginx/info/fluet-bit/README.md:1]: Will cause a merge conflict with the directory typo fix in #1470 ([commit e547dd9d](https://github.com/opensearch-project/observability/pull/1470/commits/e547dd9de896814d8693c3de9249f2ec5104c423)) can we preemptively apply the change here?

**Comment by derek-ho** [comment]
How did you test this/what is this concrete deliverable this PR is doing? I couldn't really comprehend on first pass. Also can you make sure all of the PRs you are doing are aimed at Main branch? Or at least back ported in a timely fashion? If they are not it will create a big headache with conflicts later on when other changes are needed

File: integrations/service-map/samples/preloaded/img/nginx-dashboard.png
Comment (YANG-DB) [reply integrations/service-map/samples/preloaded/img/nginx-dashboard.png:1]: its a place-holder...

File: integrations/nginx/info/fluet-bit/README.md
Comment (YANG-DB) [reply integrations/nginx/info/fluet-bit/README.md:1]: sure

**Comment by YANG-DB** [comment]
the PR is aiming for adding documentation and description sample of the services concept support for the Integration schema
In addition
 - adding fields mapping support sample (using data-prepper)
 - adding Observability Correlation using serviceName
 - sample of data-prepper dashboard for serviceMap integration

**Comment by YANG-DB** [comment]
concluded with - https://github.com/opensearch-project/observability/pull/1494
removing ..

**Pull request closed without merging**